packages/forms-web-app/src/services/interested-party.service.js

```diff
@@ -57,12 +52,20 @@ const getInterestedPartyFromSession = (req) => {
 };
 
 /**
- * confirms the interested party from session has appealReference set
+ * resets interested party in req.session
+ * @param {import('express').Request} req
+ */
+const resetInterestedPartySession = (req) => {
+	req.session.interestedParty = {};
+};
+
+/**
+ * confirms the interested party from session has caseReference set
  * @param {import('express').Request} req
  * @returns {boolean}
  */
-const confirmInterestedPartySessionAppealReference = (req) => {
-	return !!req.session.interestedParty?.appealNumber;
+const confirmInterestedPartySessionCaseReference = (req) => {
```

Comment:
thoughts on renaming these bool checks away from confirm? confirm sounds more like performing an action than checking a bool to me

Reply:
Will see if I can think of a better name, always the hardest part!